docs/lido-dao.md

```diff
@@ -1,17 +1,46 @@
+The Lido DAO is a Decentralised Autonomous Organisation that manages the liquid staking protocols by deciding on key parameters (e.g., setting fees, assigning node operators and oracles, etc.) through the voting power of governance token (`LDO`) holders. Also, the DAO will accumulate service fees and spend them on research, development, liquidity mining and protocol upgrades. 
 
-Also, the DAO will accumulate service fees and spend them on insurance, research, development, and protocol upgrades. Initial DAO members will take part in the threshold signature for Ethereum 2.0 by making BLS threshold signatures.
+## Why DAO?
 
-Since WithdrawalsManagerStub contract has been deployed, new validators withdrawal credentials point to WithdrawalsManagerStub contract controlled by Lido DAO.
-Validators before that deployment have withdrawal credentials pointing to mulsig wallet.
+The DAO is the logical compromise between full centralization and decentralisation, which allows the deployment of competitive products without full centralization and custody on the exchanges. We do not believe that it is possible to make a liquid staking protocol that is completely trustless. A DAO is an optimal structure for launching Lido as:
 
-The Lido DAO is an [Aragon organization](https://aragon.org/dao). Since Aragon provides a full end-to-end framework to build DAOs, we use its standard tools. The protocol smart contracts extend AragonApp base contract and can be managed by the DAO.
+- Lido is highly dependent on the design and restrictions of staking implementations on different chains; 
+- Staking protocols may change and therefore Lido should be upgradable; 
+- DAO governance is better than one person or a developer's team for making decisions about changes in Lido; 
+- DAO will be able to cover the costs of developing and upgrading the protocol from the DAO token treasury. 
+- Only DAO can make decisions regarding self-cover ([more](https://blog.lido.fi/offline-slashing-risks-are-self-cover-options-enough/) in blog);
 
-- [Lido DAO Mainnet](https://mainnet.lido.fi/#/lido-dao/)
-- [Lido DAO Prater Testnet](https://testnet.testnet.fi/#/lido-testnet-prater/)
+The DAO will accumulate service fees from Lido, which can be funnelled into the insurance and development funds, distributed by the DAO.
 
-## LDO Token
+## Functions
 
-LDO is a [MiniMeToken](https://github.com/Giveth/minime) granting governance rights in the Lido DAO. By holding the LDO token, one is granted voting rights within the Lido DAO. The more LDO locked in a user’s voting contract, the greater the decision-making power the voter gets.
+Lido is managed by the Lido DAO. The DAO members govern Lido to ensure its efficiency and stability. The Lido DAO should do the following:
```

Comment:
Looks like DAO here is mixed with dev and management team heavily. The main DAO means of operations are Aragon votes and easytracks and a lot of things in this list are impossible to be done by them.

Reply:
I don't make any reference here to Aragon, nor EasyTrack